src/tools/debugger/mhpdbgc.c

```diff
@@ -114,8 +114,7 @@ char loopbuf[4] = "";
 static const struct cmd_db cmdtab[] = {
    {"r0",            mhp_r0},
    {"r" ,            mhp_regs},
-   {"e",             mhp_enter},
-   {"ed",            mhp_enter},
+   {"m",             mhp_modmem},
```

Comment:
Could  you please explain a bit more about
the reason to rename? With what exactly command
was the clash?
Also maybe "poke"? :) Doesn't matter much,
"modmem" can do too.